api/src/main/java/org/pmiops/workbench/api/CohortReviewController.java

```diff
@@ -705,6 +722,9 @@ public void setUserProvider(Provider<User> userProvider) {
     cohortReview.incrementReviewedCount();
     cohortReviewService.saveCohortReview(cohortReview);
 
+    Workspace workspace = workspaceService.getRequired(workspaceNamespace, workspaceId);
+    workspaceService.updateRecentWorkspaces(workspace.getWorkspaceId(), userProvider.get().getUserId());
```

Comment:
It looks like you do this in many places. I'd make a local helper function that just takes the workspace firecloud name and namespace, since it can fetch the workspace and userid itself. So you'd just have `this.updateRecentWorkspaces(workspaceNamespace, workspaceFirecloudName)` or some such.